<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextLocationWidget</class>
 <widget class="QWidget" name="TextLocationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxLocation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Use Window Location</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonUC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Top Center</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqTopCenter32.png</normaloff>:/pqWidgets/Icons/pqTopCenter32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QToolButton" name="toolButtonUR">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Top Right Corner</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqTopRightCorner32.png</normaloff>:/pqWidgets/Icons/pqTopRightCorner32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QToolButton" name="toolButtonUL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Top Left Corner</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqTopLeftCorner32.png</normaloff>:/pqWidgets/Icons/pqTopLeftCorner32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="toolButtonLR">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Bottom Right Corner</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqBottomRightCorner32.png</normaloff>:/pqWidgets/Icons/pqBottomRightCorner32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QToolButton" name="toolButtonLL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Bottom Left Corner</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqBottomLeftCorner32.png</normaloff>:/pqWidgets/Icons/pqBottomLeftCorner32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QToolButton" name="toolButtonLC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Bottom Center</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../../Components/Resources/pqComponents.qrc">
          <normaloff>:/pqWidgets/Icons/pqBottomCenter32.png</normaloff>:/pqWidgets/Icons/pqBottomCenter32.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroupLocation</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxPosition">
     <property name="title">
      <string>Lower Left Corner</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Pos1Y">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Pos1X">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../Components/Resources/pqComponents.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupLocation"/>
 </buttongroups>
</ui>
